aries-site/src/examples/templates/content-layouts/ContentDrivenLayout.js

```diff
@@ -0,0 +1,111 @@
+import { useState } from 'react';
+import PropTypes from 'prop-types';
+import {
+  Box,
+  Button,
+  Form,
+  FormField,
+  Notification,
+  Paragraph,
+  Text,
+  TextInput,
+} from 'grommet';
+
+import { ModalBody, ModalDialog, ModalFooter } from '../ModalDialog';
+
+const path = '/products/data/47287498729/collections/KCHDvfcByKvvjymNheg';
+const parts = path.split('/').map(part => `/${part}`);
+const matchPath = [
+  ...parts.map(part => ({
+    regexp: new RegExp(part),
+    message: `Location paths do not match. Expecting ${part}.`,
+    status: 'error',
+  })),
+  {
+    regexp: new RegExp(path),
+    message: 'Location paths do not match.',
+    status: 'error',
+  },
+  function validate(value) {
+    if (value !== path) {
+      return {
+        message: `Location paths do not match. 
+Expected: ${path}
+Received: ${value}`,
+        status: 'error',
+      };
+    }
+    return undefined;
+  },
+];
+
+export const ContentDrivenLayout = ({ containerRef }) => {
+  // containerRef is for demonstration purposes on this site. Most
+  // implementations should likely remove.
+  const [showModal, setShowModal] = useState(true);
+  const [value, setValue] = useState({ deletionPath: '' });
+  const onClose = () => setShowModal(false);
+  const onChange = nextValue => {
+    setValue(nextValue);
+  };
+
+  return (
+    <Box align="center" justify="center" fill>
+      <Button
+        primary
+        label="Display Modal"
+        onClick={() => setShowModal(true)}
+      />
+      {showModal ? (
+        <ModalDialog
+          title="Delete Data"
+          onClose={onClose}
```

Comment:
We don't include the "Close" in the modal header when there are actions in the Layer. Can we remove ?
```suggestion
```

Related guidance: https://design-system.hpe.design/components/layer?q=layer#actionable

Reply:
done